spec/controllers/task_managements_controller_spec.rb

```diff
@@ -33,12 +36,13 @@
       end
       it "assigns tasks created to tasker" do
         get :index
-        expect(assigns(:tasks)).to eql @user.tasks_created.to_a
+        assign_tasks
+        expect(@tasker.tasks_created.size).to eql(2)
```

Comment:
In testing, you want to be clear about your assertions. @tasker was defined earlier in this test. What I think you want to test for is the controller assigned task instance being equal to the user tasks.